### PR TITLE
Adds styling and updates home page

### LINKED ISSUE #16: Change styling and layout
# Enhancement

#### TODO:

1. Change color scheme of phone app.
2. Import expo camera configurations for expo

### PR DESCRIPTION
**Pull request by abasnfarah**
## Describe your changes

To test locally do the following:
1. Add clerk private keys from discord to `.env` and `apps/expo/app.config.ts` files
2. Then rerun the following:
```bash
pnpm i

pnpm db-push

pnpm dev
```

To see on phone first run `pnpm dev` in the `/apps/expo` directory to get the QR code. Then close the server and re run `pnpm dev` in the root directory. 

### Changes
1. Adds the styling and logo to the home page.
2. Sets up the expo camera config for future PR's with a record function.
3. Changes colorscheme to Raeda color scheme.

#### Issues
closes #16 

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,16 +9,16 @@ node_modules
 coverage
 
 # database
-prisma/db.sqlite
```
Comment (abasnfarah) [inline .gitignore:12]: These areas in the .gitignore tell git what files to ignore. IE I want it to ignore all database files and python cache files.

File: apps/expo/package.json [hunk 2]
```diff
@@ -20,14 +20,17 @@
...
         "expo": "^48.0.6",
         "expo-auth-session": "^4.0.3",
+        "expo-camera": "^13.6.0",
         "expo-random": "^13.0.0",
         "expo-secure-store": "^12.1.1",
         "expo-status-bar": "^1.4.2",
         "expo-web-browser": "^12.1.1",
         "nativewind": "^2.0.11",
         "react": "18.2.0",
+        "react-daisyui": "^4.1.2",
```
Comment (abasnfarah) [inline apps/expo/package.json:33]: This is the component library We plan on using for our project

File: apps/expo/src/screens/home.tsx [hunk 3]
```diff
@@ -12,7 +12,7 @@ import { trpc } from "../utils/trpc";
 const SignOut = () => {
   const { signOut } = useAuth();
   return (
-    <View className="rounded-lg border-2 border-gray-500 p-4">
+    <View className="rounded-lg border-2 border-[#78b788] p-4">
```
Comment (abasnfarah) [inline apps/expo/src/screens/home.tsx:15]: Hex colors for this view component. This is how styling works in [Tailwind Css](https://tailwindcss.com/docs/border-style).

**Review by ben-bau — APPROVED** [review]
Looks good!

**Review by ben-bau — APPROVED** [review]
Looks good!

**Pull request merged**